### ISSUE TITLE
TEST GPWS Callouts

### ISSUE THREAD

**Issue by DerHofnarr** [flybywiresim/a32nx#5436]
### Aircraft Version

Development

### Description

How about adding a GPWS test to the aircraft? (butten on the left of the PFD) press it for like 3 sec. and play this sound --> https://www.youtube.com/watch?v=x-udF-J0D3k ? had it in the FSLabs320 at FSX and i think it would be a GREAT improvement of the FBW A320nx :-) love your plane <3

Cheers ~ Hofnarr 

### Discord Username (optional)

DerHofnarr#4532

**Comment by hotshotp** [comment]
The GPWS self test logic is as follows:

if you momentarily press the button, it will play a short test. If you hold the button until the glideslope callout finishes, it will do the long test. I have implemented the wwise events for the first part (gpws_test_short) and the second part (gpws_test_long). Im pretty sure @ExampleWasTaken was considering working on it. Hopefully we will have this feature some time soon!

**Comment by ExampleWasTaken** [comment]
> The GPWS self test logic is as follows:
> 
> if you momentarily press the button, it will play a short test. If you hold the button until the glideslope callout finishes, it will do the long test. I have implemented the wwise events for the first part (gpws_test_short) and the second part (gpws_test_long). Im pretty sure @ExampleWasTaken was considering working on it. Hopefully we will have this feature some time soon!

Yep, still on it, but I have a lot to do at the moment so didn't have the time to do it yet. :)

**Comment by bouveng** [comment]
It is not just playing a sound, it also shows different things on the screens and lights different buttons if I'm not mistaken.

**Comment by DerHofnarr** [comment]
> It is not just playing a sound, it also shows different things on the screens and lights different buttons if I'm not mistaken.

Thats correct it shows the terrein collors on the ND if i remember correctly

**Comment by ExampleWasTaken** [comment]
> > It is not just playing a sound, it also shows different things on the screens and lights different buttons if I'm not mistaken.
> 
> Thats correct it shows the terrein collors on the ND if i remember correctly
> 
>  Cheers ~ Hofnarr

I could be totally wrong on this, but I think it only displays on the ND when testing it through the CIDS.

**Comment by hotshotp** [comment]
> > > It is not just playing a sound, it also shows different things on the screens and lights different buttons if I'm not mistaken.
> > 
> > Thats correct it shows the terrein collors on the ND if i remember correctly
> > 
> >  Cheers ~ Hofnarr
> 
> I could be totally wrong on this, but I think it only displays on the ND when testing it through the CIDS. 

CIDS? that's for the cabin, the chimes and stuff. The button triggers the ND colors and I have references for you to send over discord. Just dm me when you need them

**Comment by tracernz** [comment]
*CFDS

**Comment by ExampleWasTaken** [comment]
> *CFDS

@hotshotp that's what I meant. thanks tracer :)
